.github/workflows/release_bump.yml

```diff
@@ -21,8 +21,31 @@ jobs:
     name: Unit Tests
     uses: ./.github/workflows/code_quality.yml
     with:
+      branch: mainline 
+  LinuxIntegrationTests:
+    name: Linux Integration Tests
```

Comment:
We should add `needs: UnitTests` here. No point in running integration tests if Unit Tests are going to fail.

Reply:
Revised and fixed